godtools/Views/Cells/LanguageTableViewCell.swift

```diff
@@ -9,13 +9,45 @@
 import UIKit
 
 class LanguageTableViewCell: UITableViewCell {
-
+    
+    @IBOutlet weak var downloadButton: UIButton!
+    @IBOutlet weak var deleteButton: UIButton!
+    @IBOutlet weak var languageLabel: GTLabel!
+    
     override func awakeFromNib() {
         super.awakeFromNib()
+        setupStyle()
     }
 
     override func setSelected(_ selected: Bool, animated: Bool) {
         super.setSelected(selected, animated: animated)
     }
     
+    // MARK: - Actions
+    
+    @IBAction func pressDownloadButton(_ sender: Any) {
+    }
+    
+    @IBAction func pressDeleteButton(_ sender: Any) {
+    }
+    
+    // MARK: - Helpers
+    
+    fileprivate func setupStyle() {
+        let selectedView = UIView()
+        selectedView.backgroundColor = UIColor.gtGreen.withAlphaComponent(0.25)
+        self.selectedBackgroundView = selectedView
+    }
+    
+    func languageExist(exist:Bool) {
```

Comment:
if you declared the function like this:
````swift
func languageExists(_ exist:Bool) {
````
entonces el llamador no tiene que escribir "exist:" porque "_" esta aqui.

````swift
cell.languageExists(exist: true)
//becomes
cell.languageExists(true)
````